---
title: 'Live-cell imaging reveals the trade-off between target search flexibility and efficiency for Cas9 and Cas12a'
date: 2024-04-22
venue: Nucleic Acid Research
authors: Lorenzo Olivi, Cleo Bagchus, Victor Pool, Ezra Bekkering, Konstantin Speckner, Hidde Offerhaus, Wen Y Wu, Martin Depken, Koen J.A. Martens, John van der Oost, Raymond H.J. Staals, Johannes Hohlbein'
paperurl: 'https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkae283/7655788'
# rgate: ''
tagline: 'Paper'
teaser: 'imgs/dCas12_2023.png'
doi: '10.1093/nar/gkae283'
---

<h2> Abstract </h2>
<p align= "justify">
CRISPR-Cas systems have widely been adopted as genome editing tools, with two frequently employed Cas nucleases being SpyCas9 and LbCas12a. Although both nucleases use RNA guides to find and cleave target DNA sites, the two enzymes differ in terms of protospacer-adjacent motif (PAM) requirements, guide architecture and cleavage mechanism. In the last years, rational engineering led to the creation of PAM-relaxed variants SpRYCas9 and impLbCas12a to broaden the targetable DNA space. By employing their catalytically inactive variants (dCas9/dCas12a), we quantified how the protein-specific characteristics impact the target search process. To allow quantification, we fused these nucleases to the photoactivatable fluorescent protein PAmCherry2.1 and performed single-particle tracking in cells of Escherichia coli. From our tracking analysis, we derived kinetic parameters for each nuclease with a non-targeting RNA guide, strongly suggesting that interrogation of DNA by LbdCas12a variants proceeds faster than that of SpydCas9. In the presence of a targeting RNA guide, both simulations and imaging of cells confirmed that LbdCas12a variants are faster and more efficient in finding a specific target site. Our work demonstrates the trade-off of relaxing PAM requirements in SpydCas9 and LbdCas12a using a powerful framework, which can be applied to other nucleases to quantify their DNA target search.